Mathlib/Tactic/Linter/DocPrime.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2024 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import Lean.Elab.Command
+
+/-!
+#  The "docPrime" linter
+
+The "docPrime" linter emits a warning on declarations that have no doc-string and whose
+name ends with a `'`.
+-/
+
+open Lean Elab
+
+namespace Mathlib.Linter
+
+/--
+The "docPrime" linter emits a warning on declarations that have no doc-string and whose
+name ends with a `'`.
+
+The file `scripts/no_lints_prime_decls.txt` records the list of exceptions at the moment in
```

Comment:
What do you think about "contains a list of temporary exceptions to this linter, which is intended to become empty over time"?